images/kube-api-proxy/pkg/rewriter/target_request.go

```diff
@@ -276,3 +242,38 @@ func (tr *TargetRequest) ResourceForLog() string {
 
 	return "UNKNOWN"
 }
+
+func shouldRewriteResource(kind string, isCore bool) bool {
```

Comment:
```suggestion
func shouldRewriteResource(kind string, isCore bool) bool {
```

It is not `kind`, it is a `resourceType`